gradient/commands/datasets.py

```diff
@@ -676,3 +685,94 @@ def update_status():
                         for pre_signed in pre_signeds:
                             update_status()
                             pool.put(self._delete, url=pre_signed.url)
+
+
+class ImportDatasetCommand(BaseDatasetsCommand):
+    def create_secret(self, key, value, expires_in=86400):
+        client = api_sdk.clients.SecretsClient(
+            api_key=self.api_key,
+            logger=self.logger,
+            ps_client_name=CLI_PS_CLIENT_NAME,
+        )
+
+        response = client.ephemeral(key, value, expires_in)
+        return response
+
+
+    def get_command_string(self, url):
+        return "%s %s /data/output" % (IMPORTER_COMMAND, url)
+
+
+    def get_command(self, url, http_auth):
+        
+        if url.scheme == 'git' or url.scheme == 's3':
```

Comment:
isn't it easier to write: command_url = url.geturl()

then do conditionals to change it and return at the end?